Created Landing Page

Drayved opened the pull request:
Created a rough draft of the Landing page. Also created a basic fetch request to the api.

Drayved committed:
Created Landing Page

Created a rough draft of the Landing page. Also created a basic fetch request to the api.

AOA19 commented on src/index.css:
Could you separate the styling for each component into their own css file and keep global styling inside of index.css? I think it would be easier to maintain that way once we all start adding our own styling.

Drayved replied:
Great idea. That would help a ton in keeping things better organized. I made those changes. Added a Landing.css, Trapezoid.css and a ConnectingLines.css. 

Please let me know if there are any other issues.

Drayved committed:
Added CSS files for each component

Created a CSS file for each component to organize better. Also added styling to the dropdowns. When hovering over it will show the full drop down.

Also fixed the border issue. Border is now responsive.
